### ISSUE TITLE
Adjust votekick system so teams can kick independently of each other

### ISSUE THREAD

**Issue by Malfunktion-issues** [mastercomfig/team-comtress-2#527]
### Description

Title says everything. Usually whenever both teams have a bot, one team calls a vote, members of that team tend to send incensed messages to the other team for "not kicking" their bot. You know what I mean, accusations of being brain dead, calls of hating fun, et cetera, et cetera. It would streamline the process and cut down on unnecessary salt.
### Checklist

- [x] None of the open *or* closed issues document this idea.
- [x] This is a new idea, not a bug.
- [ ] This idea does not seriously affect game balance.
- [ ] This is obviously an improvement, not something that people commonly disagree on.
- [x] This is not an idea for new game content (e.g. new weapons, maps, sounds).

- [x] This idea is for the code of the game, not for textures, sounds, or other resources.
- [ ] This idea has been implemented in vanilla TF2.
- [ ] There is proof that this idea is well liked (e.g. a history of use of a similar mod in a competitive league).
- [ ] This idea has been implemented in a mod. 
- [ ] This feature has existed in the vanilla game but was removed.
- [ ] This idea is for internal procedures or tooling (e.g. improving the build process, supporting new systems).

### Alternatives

1. 
2. 
3.

**Comment by treacherousfiend** [comment]
Not a bad idea... might require a lot of elbow grease to work, but I'll look into it.

**Comment by MeDerpWasTaken** [comment]
This could lead to problem with toxicity, but for now I think it's important

**Comment by RealHoriuchi** [comment]
I feel like there is a lot of potential abuse with enemy votekicks, there is likely a good reason it isn't already in TF2.

**Comment by treacherousfiend** [comment]
Ah, I misread the issue, it was late when I read it... oops. Anyway, I assumed that this issue was for fixing the issue of not being able to call a votekick on your team because, unbeknownst to you, the enemy team _also_ had a votekick in progress.

I will _NOT_  be looking into this issue, since it seems like more of a valve thing.
I _will_ however, look into what I assumed this issue was, as that seems more like a bugfix.

**Comment by Malfunktion-issues** [comment]
> Ah, I misread the issue, it was late when I read it... oops. Anyway, I assumed that this issue was for fixing the issue of not being able to call a votekick on your team because, unbeknownst to you, the enemy team _also_ had a votekick in progress.
> 
> I will _NOT_ be looking into this issue, since it seems like more of a valve thing.
> I _will_ however, look into what I assumed this issue was, as that seems more like a bugfix.

No, no, you had it right the first time, my suggestion is simply allowing teams to call votes for their own members independently of each other. Blue can only kick Blue, Red can only kick Red. Even if we weren't in the midst of a malicious bot problem, the ability to kick enemies would be an incredibly dumb addition.

**Comment by treacherousfiend** [comment]
I see, I think it was just that the wording in the original post was taken the wrong way.

**Comment by Malfunktion-issues** [comment]
Yeah, I probably could have worded it better, I was running on fumes at that point.

**Comment by RealHoriuchi** [comment]
> > Ah, I misread the issue, it was late when I read it... oops. Anyway, I assumed that this issue was for fixing the issue of not being able to call a votekick on your team because, unbeknownst to you, the enemy team _also_ had a votekick in progress.
> > I will _NOT_ be looking into this issue, since it seems like more of a valve thing.
> > I _will_ however, look into what I assumed this issue was, as that seems more like a bugfix.
> 
> No, no, you had it right the first time, my suggestion is simply allowing teams to call votes for their own members independently of each other. Blue can only kick Blue, Red can only kick Red, but now at the same time. Even if we weren't in the midst of a malicious bot problem, the ability to kick enemies would be an incredibly dumb addition.

This along with a popup that says a vote has started on the enemy team to X reason would make this a fantastic change.

**Comment by Malfunktion-issues** [comment]
> This along with a popup that says a vote has started on the enemy team to X reason would make this a fantastic change.

God yes, anything is better than the stock "You cannot call a vote while another is in progress" or whatever message.

@treacherousfiend I should probably make another issue for this but would changing the minimum player participation for a vote be possible? Right now it takes at least six people to participate in a vote for one to not automatically fail (for example a four to one vote to kick will always come back as a fail but a four to two will succeed).

**Comment by treacherousfiend** [comment]
make a different issue for it, but in theory it should be relatively easy to make minimum players for voting maybe be a percentage?

**Comment by tauin** [comment]
is this needed anymore? seems to be "fixed" in the latest patche(s)

**Comment by micwoj92** [comment]
its still not fixed in tc2
